README.md

```diff
@@ -34,7 +34,10 @@ translation:
     msgid "Select {} to change"
     msgstr "Wybierz {.accusative} do zmiany"
 
-Languages that use grammatical cases of nouns are: German, Polish, Czech.
+Languages that use grammatical cases of nouns are: Armenian, Assamese, most 
```

Comment:
```suggestion
Some of the languages that use grammatical cases for nouns are: Armenian, Assamese, most 
```

Reply:
not to put too fine a point but I'd refrain from such absolute statements. from the top of my head Greek should be here too :p

Reply:
Good point. Got the list from Wikipedia article (will try to verify Greek then :)